_studio/shared/umc/codec/jpeg_enc/src/jpegenc.cpp

```diff
@@ -696,6 +696,9 @@ JERRCODE CJPEGEncoder::SetParams(
   {
     for(c = 0; c < m_jpeg_ncomp; c++)
     {
+      if (c >= MAX_HUFF_TABLES)
```

Comment:
definitely no need to check inside the loop. Also, is it possible that ```m_jpeg_ncomp >= MAX_HUFF_TABLES```?

Reply:
Agree, there is definitely no need to check inside the loop. I changed commit to
```suggestion
      if (c >= MAX_HUFF_TABLES)
        return JPEG_ERR_PARAMS;
      for(c = 0; c < m_jpeg_ncomp; c++)
      {
```
It's possible that `m_jpeg_ncomp >= MAX_HUFF_TABLES ` because it is limited by another constant `MAX_COMPS_PER_SCAN`

Reply:
ok